instrumentation/jsonrpc4j-1.3/library/src/main/java/io/opentelemetry/instrumentation/jsonrpc4j/v1_3/JsonRpcServerAttributesExtractor.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.jsonrpc4j.v1_3;
+
+import com.googlecode.jsonrpc4j.AnnotationsErrorResolver;
+import com.googlecode.jsonrpc4j.DefaultErrorResolver;
+import com.googlecode.jsonrpc4j.ErrorResolver;
+import com.googlecode.jsonrpc4j.MultipleErrorResolver;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.AttributesBuilder;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.instrumentation.api.instrumenter.AttributesExtractor;
+import javax.annotation.Nullable;
+
+// Check https://opentelemetry.io/docs/specs/semconv/rpc/json-rpc/
+final class JsonRpcServerAttributesExtractor
+    implements AttributesExtractor<JsonRpcServerRequest, JsonRpcServerResponse> {
+
+  private static final AttributeKey<Long> RPC_JSONRPC_ERROR_CODE =
```

Comment:
usually we add a comment like `// copied from RpcIncubatingAttributes` for incubating attributes that we copy